Libplanet/Store/StateStoreExtensions.cs

```diff
@@ -105,5 +106,34 @@ IReadOnlyList<string> rawStateKeys
             HashDigest<SHA256> stateRootHash
         ) =>
             stateStore.GetStateRoot(stateRootHash).Recorded;
+
+        public static IAccount GetAccount(
+            this IStateStore stateStore,
+            HashDigest<SHA256>? stateRootHash,
+            string key
+        )
+        {
+            ITrie trie = stateStore.GetStateRoot(stateRootHash);
+            if (trie.Get(new[] { EncodeKey(key) })[0] is Bencodex.Types.List rawAddress)
```

Comment:
I guess It may throw NRE when there is no entity for the given `key`.

Reply:
So, should we support `CreateAccount` in `StateStoreExtension`?